.travis.yml

```diff
@@ -1,14 +1,21 @@
 os:
-  - linux
-
+- linux
 language: go
-
 go:
-  - 1.9
-
+- 1.9
 services:
-  - docker
-
+- docker
 script:
-  - CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o certificate-controller .
-  - docker build -t certificate-controller:1.0 .
+- CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o certificate-controller
+  .
+- docker build -t easymile/certificate-controller:1.0-$(echo $TRAVIS_COMMIT | cut -c1-7) .
```

Comment:
"echo ${TRAVIS_COMMIT:0:7}" is better.
And it would be better to store it in a variable then use it everywhere.